scopes/api.py

```diff
@@ -0,0 +1,90 @@
+from django.conf import settings
+from django.utils import translation
+from django.utils.translation.trans_real import translation as trans_real_translation
+from rest_framework import serializers
+from rest_framework.schemas import AutoSchema
+from rest_framework.views import APIView
+
+from oidc_apis.models import ApiScope
+from oidc_apis.scopes import CombinedScopeClaims
+from tunnistamo.pagination import DefaultPagination
+from tunnistamo.utils import TranslatableSerializer
+
+ENGLISH_LANGUAGE_CODE = 'en'
+LANGUAGE_CODES = [l[0] for l in settings.LANGUAGES]
+assert ENGLISH_LANGUAGE_CODE in LANGUAGE_CODES
```

Comment:
Importing will fail if english is not supported?

Reply:
Well, the endpoint won't work if english isn't supported, so I figured it would be better to fail soon. How would you like it to work?